packages/features/insights/server/events.ts

```diff
@@ -442,6 +444,7 @@ class EventsInsights {
 
       return {
         ...bookingTimeStatus,
+        bookerNoShow: booker.noShow,
```

Comment:
can we call this noShowGuest? to be consistent with

- No-Show (Guests)
- No-Show (Hosts)